setup.py

```diff
@@ -20,12 +20,10 @@ def get_version():
   import sys
   assert sys.version_info[0] > 2, "Python 2 is no longer supported!"
 
-  result = "?"
-  with open("pymtl3/__init__.py") as f:
-    for line in f:
-      if line.startswith("__version__"):
-        _, result, _ = line.split('"')
-  return result
+  _d = {}
+  with open(path.join(path.dirname(__file__), "pymtl3/version.py")) as f:
```

Comment:
Do other projects also use version.py?

Reply:
Hypothesis and mypy both use `version.py`